runtime/doc/eval.txt

```diff
@@ -2824,8 +2826,8 @@ char2nr({expr}[, {utf8}])					*char2nr()*
 			char2nr("ABC")		returns 65
 <		When {utf8} is omitted or zero, the current 'encoding' is used.
 		Example for "utf-8": >
-			char2nr("�")		returns 225
-			char2nr("�"[0])		returns 195
+			char2nr("?")		returns 225
+			char2nr("?"[0])		returns 195
```

Comment:
This change is not needed.
It's better to set `enc=utf-8` when you edit `eval.txt`.